element_array_ephys/probe.py

```diff
@@ -177,33 +84,20 @@ class Probe(dj.Lookup):
 
 @schema
 class ElectrodeConfig(dj.Lookup):
-    """Electrode configuration setting on a probe.
-
-    Attributes:
-        electrode_config_hash (foreign key, uuid): unique index for electrode configuration.
-        ProbeType (dict): ProbeType entry.
-        electrode_config_name ( varchar(4000) ): User-friendly name for this electrode configuration.
-    """
-
```

Comment:
Should we keep the docstrings?

Reply:
I'll remove them.

Reply:
@JaerongA Sorry, I am not quite understanding.  Why where the docstrings removed?